modules/manage/examples/kubernetes/user-crds.feature

```diff
@@ -0,0 +1,86 @@
+@cluster:sasl
+Feature: User CRDs
+  Background: Cluster available
+    Given cluster "sasl" is available
+
+  @skip:gke @skip:aks @skip:eks
+  Scenario: Manage users
+    Given there is no user "bob" in cluster "sasl"
+    And there is no user "james" in cluster "sasl"
+    And there is no user "alice" in cluster "sasl"
+    When I create CRD-based users for cluster "sasl":
+      | name  | password | mechanism     | acls |
+      | bob   |          | SCRAM-SHA-256 |      |
+      | james |          | SCRAM-SHA-512 |      |
+      | alice | qwerty   | SCRAM-SHA-512 |      |
+    Then "bob" should exist and be able to authenticate to the "sasl" cluster
+    And "james" should exist and be able to authenticate to the "sasl" cluster
+    And "alice" should exist and be able to authenticate to the "sasl" cluster
+
+  @skip:gke @skip:aks @skip:eks
+  Scenario: Manage authentication-only users
+    Given there is no user "jason" in cluster "sasl"
+    And there are already the following ACLs in cluster "sasl":
+      | user   | acls |
+      | jason  | [{"type":"allow","resource":{"type":"cluster"},"operations":["Read"]}] |
+    When I apply Kubernetes manifest:
+    """
+    # tag::manage-authn-only-manifest[]
```

Comment:
Just making sure of how these are parsed. We really only care about the manifest examples, right? Just b/c the rest of the test code/scenarios are pretty much irrelevant to the docs.

Reply:
Docs pull in anything between the tags. We're using tags for this reason: to avoid pulling in the irrelevant parts.